controllers/apim/ratelimitpolicy_controller.go

```diff
@@ -85,15 +101,338 @@ func (r *RateLimitPolicyReconciler) Reconcile(eventCtx context.Context, req ctrl
 		}
 	}
 
-	specResult, specErr := r.reconcileSpec(ctx, &rlp)
-	logger.Info("spec reconcile done", "result", specResult, "error", specErr)
-	if specErr == nil && specResult.Requeue {
-		logger.Info("Reconciling not finished. Requeueing.")
-		return specResult, nil
+	if err := r.reconcileLimits(ctx, &rlp); err != nil {
+		return ctrl.Result{}, err
+	}
+
+	vsName, ok := rlp.Annotations[RLPAnnotationVSName]
+	if !ok {
```

Comment:
Does this need to be removed?

Reply:
I guess that's the old commit, I cannot find that in the new commit.

Reply:
Okay, that constant was still present, removed.